beets/ui/__init__.py

```diff
@@ -471,29 +477,23 @@ def get_replacements():
     return replacements
 
 
-def _pick_format(album, fmt=None):
-    """Pick a format string for printing Album or Item objects,
-    falling back to config options and defaults.
-    """
+def format_(obj, fmt):
+    """Print a object, intended for Album and Item
+
+    This is equivalent to `format`, but the spec can be None
+    `fmt` is mandatory for otherwise one can just call `format(my_object)`"""
```

Comment:
I wish... however
```
>>> import beets.library
>>> i = beets.library.Item()
>>> format(i, None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: format expects arg 2 to be string or unicode, not NoneType
```
`format()` checks `spec` against None, so it does not work. This is exactly what `format_()` does: allowing a None format spec.